src/utils/uppy.ts

```diff
@@ -47,6 +47,7 @@ const configureUppy = ({
     endpoint: `${apiHost}/${API_ROUTES.buildUploadFilesRoute(itemId)}`,
     withCredentials: true,
     formData: true,
+    allowedMetaFields: [],
```

Comment:
This prop tells Uppy that the specified fields are passed to the rest of your implementation, I am not sure that you want to use an empty array for this.

Reply:
I also doubt that this is the best approach. However, this how it was implemented initially. I don't know enough about how the files are handled and what should be the right meta fields to allow. Let's discuss that.